R/utils.R

```diff
@@ -1431,6 +1431,12 @@ wrapFunctionLabel <- function(func, name, ..stacktraceon = FALSE, dots = TRUE) {
   if (name == "name" || name == "func" || name == "relabelWrapper") {
     stop("Invalid name for wrapFunctionLabel: ", name)
   }
+  if (nchar(name, "bytes") > 10000) {
```

Comment:
I'm not sure if this is actually relevant (can `name` be `NULL`/`NA` at this point?), but seems like a good idea to be defensive to those cases?

```suggestion
  if (isTRUE(nchar(name, "bytes") > 10000)) {
```